pkg/engine/lifecycletest/testdata/output/TestExternalRefresh/0-0-0/eventstream.json

```diff
@@ -1,7 +1,7 @@
 {"sequence":0,"timestamp":0,"preludeEvent":{"config":{}}}
 {"sequence":0,"timestamp":0,"resourcePreEvent":{"metadata":{"op":"create","urn":"urn:pulumi:test::test::pulumi:providers:pkgA::default","type":"pulumi:providers:pkgA","old":null,"new":{"type":"pulumi:providers:pkgA","urn":"urn:pulumi:test::test::pulumi:providers:pkgA::default","custom":true,"id":"","parent":"","inputs":{},"outputs":{},"provider":""},"detailedDiff":null,"logical":true,"provider":""}}}
-{"sequence":0,"timestamp":0,"resOutputsEvent":{"metadata":{"op":"create","urn":"urn:pulumi:test::test::pulumi:providers:pkgA::default","type":"pulumi:providers:pkgA","old":null,"new":{"type":"pulumi:providers:pkgA","urn":"urn:pulumi:test::test::pulumi:providers:pkgA::default","custom":true,"id":"68624099-2277-48e3-8b9e-a05724cb592a","parent":"","inputs":{},"outputs":{},"provider":""},"detailedDiff":null,"logical":true,"provider":""}}}
```

Comment:
Spurious diff because provider IDs are random. You could revert these snapshots in pkg/engine/lifecycletest/testdata/output/TestExternalRefresh/